syntax = "proto3";

package sentry_protos.snuba.v1;

import "google/protobuf/timestamp.proto";

message RequestMeta {
  uint64 organization_id = 1;
  string cogs_category = 2;
  string referrer = 3;
  // can be empty
  repeated uint64 project_ids = 4;
  google.protobuf.Timestamp start_timestamp = 5;
  google.protobuf.Timestamp end_timestamp = 6;
  TraceItemName trace_item_name = 7;
}

enum TraceItemName {
  TRACE_ITEM_NAME_UNSPECIFIED = 0;
  TRACE_ITEM_NAME_EAP_SPANS = 1;
}
